config/app/manifest/en_US_1.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Why does this file exist here? If it's created by your test case then you don't have the virtual filesystem configured correctly.

Why does it have `_1.json`? 

You need another test case that tests when a manifest file for a given local already exists a warning is printed to the console.